cocotb/share/lib/fli/FliImpl.cpp

```diff
@@ -751,7 +751,7 @@ GpiIterator::Status FliIterator::next_handle(std::string &name, GpiObjHdl **hdl,
         return GpiIterator::END;
     }
 
-    char *c_name;
+    char *c_name = NULL;
```

Comment:
Would it be clearer to put this in the `default` below instead?

Reply:
Uninitialized variables are code smells IMO. The code base is littered with them, but that's a problem for another day.

Reply:
They're a feature once checked by a compiler - if we were to add a case to the switch below that forgets to set c_name, then either:
* This line exists, the compiler does not find our mistake
* This line does not exist, the compiler warns that c_name is used initialized

The second case seems like an obvious win to me.